### ISSUE TITLE
Confirm dialogue doesn't work with non latin alphabet characters

### ISSUE THREAD

**Issue by KPidS** [console-rs/dialoguer#256]
I'm using Linux, Pop OS. If I change the keyboard layout to Russian, Ukrainian or Japanese, the confirm dialogue doesn't work.

This can be fixed by checking for a physical key, rather than a literal character. But as far as I've researched this is not a trivial thing to do. I'm not sure how to approach this issue with all the different languages and layouts in mind.

There is also a band aid solution. Just check against multiple characters from different languages.

**Comment by Gordon01** [comment]
Yeah, I don't like this either. But most of the CLI tools are like this. `htop` is a good example.
Getting a scancode is possible, but it would require support from `console`. https://docs.rs/console/latest/console/struct.Term.html

Which is also possible, but I came to an alternative solution:
What if we offer backup keys (like 1 / 0) if we detect a non-ascii char scanned?

So add here: https://github.com/console-rs/dialoguer/blob/master/src/prompts/confirm.rs#L181-L205
```rust
Key::Char(c) if !c.is_ascii() => { offer_10(); }
```

**Comment by pksunkara** [comment]
@Gordon01 I am unsure how #279 fixes this. Did I miss something?

**Comment by Gordon01** [comment]
Alternative keys 1 and 0 would work on any keyboard layout. I don't have any other ideas on how to make confirm dialog work on non-latin keyboards 😔

**Comment by pksunkara** [comment]
That makes sense, I missed the "keyboard" in the issue description.
